File: OSCommon/org/optimizationservices/oscommon/datastructure/osprocess/ProcessHeader.cs
using System;

namespace org.optimizationservices.oscommon.datastructure.osprocess{
	/// <summary>
	/// A data structure class that corresponds to an xml element in the OSpL schema. 
	/// @author Jun Ma	
	/// @version 1.0, 09/01/2005
	/// @since OS 1.0
	/// @copyright (c) 2005
	/// </summary>
	public class ProcessHeader{
		
		public ProcessRequest request = null;

		public ProcessResponse response = null;
	
		public string serviceURI = "";

		public string serviceName = "";
	
		public DateTime time = new DateTime(1970, 1, 1, 0, 0, 0);

		public string message = "";

	}//class ProcessHeader
}//namespace

File: OSCommon/src/org/optimizationservices/oscommon/datastructure/osprocess/ProcessHeader.java
/**
 * @(#)ProcessHeader 1.0 03/14/2004
 *
 * Copyright (c) 2004
 */
package org.optimizationservices.oscommon.datastructure.osprocess;

import java.util.GregorianCalendar;

/**
*
* <p>A data structure class that corresponds to an xml element in the OSeL schema.  
*  
* </p>
* @author Robert Fourer, Jun Ma, Kipp Martin
* @version 1.0, 03/14/2004
* @since OS 1.0
*/
public class ProcessHeader {

	public ProcessRequest request = null;

	public ProcessResponse response = null;
	
	public String serviceURI = "";

	public String serviceName = "";
	
	public GregorianCalendar time = null;

	public String message = "";

}//ProcessHeader
